trusted_applet/main.go

```diff
@@ -78,6 +79,20 @@ var (
 	persistence *storage.SlotPersistence
 )
 
+var (
+	doOnce                       sync.Once
+	counterWitnessStarted        monitoring.Counter
+	counterFirmwareUpdateAttempt monitoring.Counter
+)
+
+func initMetrics() {
+	doOnce.Do(func() {
+		mf := monitoring.GetMetricFactory()
+		counterWitnessStarted = mf.NewCounter("witness_started", "Number of times the witness was started")
+		counterFirmwareUpdateAttempt = mf.NewCounter("firmware_update_attempt", "Number of times the updater ran to check if firmware could be updated")
```

Comment:
How about `num_firmware_update_checks`? Then we probably want a `num_firmware_update_attempts` success/fail to detect problems?

Reply:
Added a success counter.